Sources/HaCWebsiteLib/Views/Workshops/WorkshopsIndexPage/WorkshopsIndexArchive.swift

```diff
@@ -0,0 +1,21 @@
+import HaCTML
+
+/// Above-the-fold for the WorkshopsIndexPage
+struct WorkshopsIndexArchive: Nodeable {
+  let workshops: [NewWorkshop]
+
+  var node: Node {
+    return El.Div[Attr.className => "WorkshopsIndexPage__archive"].containing(
+      El.H2[Attr.className => "Text--sectionHeading"].containing("Browse Workshops"),
+      El.Ul[Attr.className => "WorkshopsIndexPage__archive__list"].containing(
+        workshops.map { workshop in 
+          El.Li.containing(
+            El.A[Attr.href => "/beta/workshops/\(workshop.workshopId)", Attr.className => "WorkshopsIndexPage__archive__list__item"].containing( 
```

Comment:
We should probably stop hard-coding our URLs so if they change (as they will with `/beta/workshop`), we don't miss them.

Not a blocker, but something that is as easy as having one `WebsiteLink` struct with static functions could work (there are more elegant solutions).

Reply:
Was thinking about how best to do this. I figured it wasn't a priority for now